homeassistant/components/ffmpeg.py

```diff
@@ -96,12 +98,46 @@ def __init__(self, hass, ffmpeg_bin):
         self.hass = hass
         self._cache = {}
         self._bin = ffmpeg_bin
+        self._version = None
 
     @property
     def binary(self):
         """Return ffmpeg binary from config."""
         return self._bin
 
+    async def async_get_version(self):
+        """Return ffmpeg version."""
+        if self._version is None:
+            proc = await asyncio.create_subprocess_exec(
+                self._bin, "-version",
+                loop=self.hass.loop,
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE)
+
+            first_line = (await proc.stdout.readline()).decode()
+            _LOGGER.debug(first_line)
+
+            result = re.search(r"ffmpeg version (\S*)", first_line)
+            if result is not None:
+                self._version = result.group(0)
```

Comment:
group(0) is still used here, result.group(1) might be more logical here also (although the system works like this too)